pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -94,6 +97,32 @@ func (m *OSDHealthMonitor) Update(removeOSDsIfOUTAndSafeToRemove bool) {
 
 // checkOSDHealth takes action when needed if the OSDs are not healthy
 func (m *OSDHealthMonitor) checkOSDHealth() error {
+	err := m.checkByOSDDump()
+	if err != nil {
+		return err
+	}
+	err = m.updateCR()
```

Comment:
Since it's just checking device classes right now, let's name it more specifically
```suggestion
	err = m.checkDeviceClasses()
```

Reply:
Fixed